liblocation/src/main/java/com/mapbox/android/core/location/LocationEngine.java

```diff
@@ -16,7 +16,7 @@
  */
 public abstract class LocationEngine {
   public enum Type {
-    GOOGLE_PLAY_SERVICES, ANDROID, REPLAY
+    GOOGLE_PLAY_SERVICES, ANDROID, MOCK, REPLAY
```

Comment:
I think we should mark `MOCK` as Deprecated. That way we can internally mark it for removal at the next major release and flag to others that it's going away.